packages/aws-cdk/test/api/hotswap/fake-cloudformation-stack.ts

```diff
@@ -1,6 +1,6 @@
 import { CloudFormation } from 'aws-sdk';
```

Comment:
I would leave this file where it is - it's not really tied to hotswapping in any way, it can be used in other tests as well.